src/Mapping/Annotation/AggregateVersioning.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Gedmo\Mapping\Annotation;
+
+use Doctrine\Common\Annotations\Annotation;
+
+/**
+ * @Annotation
+ * @Target("CLASS")
+ *
+ * @author Maksim Vorozhtsov <myks1992@mail.ru>
+ */
+final class AggregateVersioning extends Annotation
```

Comment:
IMHO we should add attribute support too, at some point support for annotation will be dropped.